### ISSUE TITLE
[Accessibility] Focus after clicking the participants button

### ISSUE THREAD

**Issue by emrahcom** [jitsi/jitsi-meet#17368]
### What happened?

Our company had an accessibility scan done for Jitsi. This is the description of one problem in the accessibility report:

> When the participants menu is opened, the focus is not set into the menu. Instead it moves in the background for some TAB steps before it gets to the menu. When the menu is opened, the focus should be set into the menu. When it is closed, focus should be set onto the triggering element. 

### Platform

- [x] Chrome (or Chromium based)
- [ ] Firefox
- [ ] Safari
- [ ] Other desktop browser
- [ ] Android browser
- [ ] iOS browser
- [ ] Electron app
- [ ] Android mobile app
- [ ] iOS mobile app
- [ ] Custom app using a mobile SDK

### Browser / app / sdk version

Any browser

### Relevant log output

```shell

```

### Reproducibility

- [x] The problem is reproducible on meet.jit.si

### More details?

<img width="966" height="468" alt="Image" src="https://github.com/user-attachments/assets/2dfcbe17-3b92-4c20-bd66-2151fc806b6c" />
